### DISCUSSION TITLE
vip/%E6%8E%88%E6%9D%83%E8%B4%B9%E7%94%A8

### DISCUSSION THREAD

**Discussion by giscus** [zuihou/zuihou-admin-cloud#338] [企业版]
# vip/%E6%8E%88%E6%9D%83%E8%B4%B9%E7%94%A8

微服务中后台快速开发平台

https://tangyh.top/vip/%E6%8E%88%E6%9D%83%E8%B4%B9%E7%94%A8.html

**Comment by Johnnyshyzt** [comment]
企业版感觉还是少了点东西
1、bpm 审批流
2、自定义打印模版

一般购买企业版都是要做成产品，少了这两个功能不行的。

**Reply by zuihou** [reply]
好的
